cmsplugin_cascade/templates/cascade/bootstrap3/accordion.html

```diff
@@ -8,7 +8,7 @@
 	<div class="panel {{ panel_type }}">
 		<div class="panel-heading">
 			<{{ heading_tag }} class="panel-title">
-				<a data-toggle="collapse" data-parent="#cmsplugin_{{ instance.id }}" href="#cmsplugin_{{ instance.id }}_{{ forloop.counter0 }}">{{ panel_title }}</a>
+				<a data-toggle="collapse" {% if instance.glossary.close_others %}true{% else %}false{% endif %} data-parent="#cmsplugin_{{ instance.id }}" href="#cmsplugin_{{ instance.id }}_{{ forloop.counter0 }}">{{ panel_title }}</a>
```

Comment:
could you please point out in the Bootstrap3 docs, where the ``true`` and ``false`` attributes are mentioned.
Also, you should rather use the template filer ``...|yesno:"true,false"``.